requirements/base.txt

```diff
@@ -18,7 +18,7 @@ Markdown==3.2.1
 reportlab==3.5.6
 requests
 requests_oauthlib
-Pillow==4.0.0
+Pillow==8.2.0
```

Comment:
Question: did we test the pillow version bump ? I wasn't able to test this out locally which is why I was hesitating bumping this up

Reply:
I tested this on the "staging" that I deployed with ansible.